steam_wrapper/steam_wrapper.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python3
+import os
+import os.path
+import sys
+import shutil
+import errno
+import fnmatch
+import subprocess
+
+STEAM_PATH = "/app/bin/steam"
+STEAM_ROOT = os.path.expandvars("$HOME/.var/app/com.valvesoftware.Steam")
+
+def prompt():
+    subprocess.check_call(["zenity", "--question",
```

Comment:
Could handle this better than just letting it raise?

Reply:
Sure, no problem. What should it do, skip migrations and start Steam?

Reply:
Another thing, should we prompt for migration on next startup or record somewhere that we already asked and user said no?